Backend/app/rider-platform/rider-app/Main/src/Storage/Queries/Transformers/ServicePeopleCategory.hs

```diff
@@ -0,0 +1,25 @@
+{-# OPTIONS_GHC -Wno-orphans #-}
+{-# OPTIONS_GHC -Wno-unused-imports #-}
+
+module Storage.Queries.Transformers.ServicePeopleCategory where
+
+import qualified Data.Aeson
+import qualified Domain.Types.ServicePeopleCategory
+import Kernel.Beam.Functions
+import Kernel.External.Encryption
+import Kernel.Prelude
+import qualified Kernel.Prelude
+import Kernel.Types.Error
+import Kernel.Utils.Common (CacheFlow, EsqDBFlow, MonadFlow, fromMaybeM, getCurrentTime)
+
+convertCancellationChargesToTable :: (Kernel.Prelude.Maybe [Domain.Types.ServicePeopleCategory.CancellationCharge] -> Kernel.Prelude.Maybe Data.Aeson.Value)
+convertCancellationChargesToTable mbCancelCharge = Data.Aeson.toJSON <$> mbCancelCharge
+
+getCancellationChargesFromTable :: (Kernel.Prelude.Maybe Data.Aeson.Value -> Kernel.Prelude.Maybe [Domain.Types.ServicePeopleCategory.CancellationCharge])
+getCancellationChargesFromTable = maybe Nothing valueToMaybe
+
+valueToMaybe :: Data.Aeson.Value -> Kernel.Prelude.Maybe [Domain.Types.ServicePeopleCategory.CancellationCharge]
```

Comment:
not related to your PR, but we have this func duplicated 7 times, need to move to Kernel